impress-org/donation-block-for-stripe
# Add Google ReCaptcha v2 integration

## Description



This PR adds a new integration with Google ReCAPTCHA v2.

## Affects



Step 1 of the donation form. If integrated, you'll see a new ReCAPTCHA that when completed will allow you to move to step 2 of the donation form.

## Visuals



Please watch this video overview:

https://www.loom.com/share/24bda48da6c24abfaacaa503043c07b8

## Testing Instructions



1. Install the current block from the WP.org directory
2. Connect Stripe and set up multiple donation forms on the same page with different settings per block
3. Install the new version of the plugin via zip
4. Compare before and after for any breakage
5. Activate the new ReCAPTCHA integration
6. Ensure you're unable to donate without first confirming the captcha.



---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1203555946043643
